### PR TITLE
L1 Test code cleanup

### PR DESCRIPTION
**Pull request by akhilbhas**
Cleanup of unnecessary code in L1 tests

### CODE REVIEW

File: src/test_l1_hdmi_cec_driver.c [hunk 1]
```diff
@@ -832,7 +736,7 @@ void test_hdmicec_hal_l1_removeLogicalAddress_negative( void )
  */
 void test_hdmicec_hal_l1_removeLogicalAddress_positive( void )
 {
-    int result;
+    HDMI_CEC_STATUS result = HDMI_CEC_IO_SUCCESS;;
```
Comment (srinivasgtl) [inline src/test_l1_hdmi_cec_driver.c:739]: In the description fo this function, just make it clear that it is only for sink devices and not for the source devices.

File: src/test_l1_hdmi_cec_driver.c [hunk 2]
```diff
@@ -1342,7 +1246,7 @@ void test_hdmicec_hal_l1_setTxCallback_positive( void )
  */
 void test_hdmicec_hal_l1_hdmiCecTx_sinkDevice_negative( void )
 {
-    int result=HDMI_CEC_IO_SENT_AND_ACKD;
+    HDMI_CEC_STATUS result =HDMI_CEC_IO_SENT_AND_ACKD;
```
Comment (srinivasgtl) [inline src/test_l1_hdmi_cec_driver.c:1249]: in the below lines, where ever we have /*invalid inputs */, can we specify what is the invalid param here?

**Review by srinivasgtl — COMMENTED** [review]
1. For the Add and Remove logical address, when called from the source device point of view, we need to check the return values HDMI_CEC_IO_OPERATION_NOT_SUPPORTED. If this is not mentioned in the header file, then we will discuss this with the concerned teams.

File: src/test_l1_hdmi_cec_driver.c [hunk 3]
```diff
@@ -177,48 +160,13 @@ void DriverReceiveCallback(int handle, void *callbackData, unsigned char *buf, i
  * @param callbackData callback data passed
  * @param result async send status
  */
-void DriverTransmitCallback(int handle, void *callbackData, int result)
+void DriverTransmitCallback(int handle, void *callbackData, int result )
```
[the remark this answers is not in the document]
Comment (akhilbhas) [reply src/test_l1_hdmi_cec_driver.c:163]: note is already present in description

File: src/test_l1_hdmi_cec_driver.c (hunk 1 above)
Comment (akhilbhas) [reply src/test_l1_hdmi_cec_driver.c:739]: It is already mentioned in the Test Description.

Comment (srinivasgtl) [reply src/test_l1_hdmi_cec_driver.c:695]: Like in line 597, please mention the same around line 673.  The statement is " This test case is only applicable for sink devices"

File: src/test_l1_hdmi_cec_driver.c [hunk 4]
```diff
@@ -600,7 +504,7 @@ void test_hdmicec_hal_l1_getPhysicalAddress_positive( void )
  */
 void test_hdmicec_hal_l1_addLogicalAddress_negative( void )
 {
-    int result;
+    HDMI_CEC_STATUS result = HDMI_CEC_IO_SUCCESS;;
```
Comment (srinivasgtl) [inline src/test_l1_hdmi_cec_driver.c:507]: For all Add and remove logic address positive or negative functions, we need to check for NOT_SUPPORTED error code on all the source devices

File: src/test_l1_hdmi_cec_driver.c [hunk 5]
```diff
@@ -1135,7 +995,7 @@ void test_hdmicec_hal_l1_getLogicalAddress_sourceDevice_positive( void )
  */
 void test_hdmicec_hal_l1_setRxCallback_negative ( void )
 {
-    int result;
+    HDMI_CEC_STATUS result = HDMI_CEC_IO_SUCCESS;;
```
Comment (srinivasgtl) [inline src/test_l1_hdmi_cec_driver.c:998]: at line 1020, change the comment.

File: src/test_l1_hdmi_cec_driver.c (hunk 4 above)
Comment (akhilbhas) [reply src/test_l1_hdmi_cec_driver.c:507]: Addressed the review

File: src/test_l1_hdmi_cec_driver.c (hunk 2 above)
Comment (akhilbhas) [reply src/test_l1_hdmi_cec_driver.c:1281]: Addressed the review

File: src/test_l1_hdmi_cec_driver.c (hunk 1 above)
Comment (akhilbhas) [reply src/test_l1_hdmi_cec_driver.c:771]: Addressed

File: src/test_l1_hdmi_cec_driver.c (hunk 5 above)
Comment (srinivasgtl) [reply src/test_l1_hdmi_cec_driver.c:1062]: @akhilbhas ,  This GetLogicalAddress is applicable for both source and sink devices.  This is common as the source devices gets their logical address from Open() and then provide the same through this API.

Comment (akhilbhas) [reply src/test_l1_hdmi_cec_driver.c]: updated the descriptions

**Review by srinivasgtl — DISMISSED** [review]

**Review by ishthiyaqahmed — APPROVED** [review]

**Review by srinivasgtl — APPROVED** [review]
approved

**Pull request merged**